app/components/map/index.js

```diff
@@ -423,13 +395,12 @@ class Map extends Component {
             provider={MapView.PROVIDER_GOOGLE}
             mapType="hybrid"
             rotateEnabled={false}
-            onPress={this.onMapPress}
             initialRegion={this.state.region}
-            onRegionChange={debounce(region => this.updateRegion(region), 100)}
-            onRegionChangeComplete={this.updateRegion}
+            onRegionChangeComplete={region => this.updateRegion(region)}
```

Comment:
You can use this.updateRegion directly as the function so we avoid creating an unnecessary function.